### PR TITLE
Add marquee to stream bar on front page

### PR DESCRIPTION
**Pull request by jdliaw**
Issue: https://github.com/uclaradio/uclaradio/issues/108
* used CSS3 marquee animation
* moves linearly left to right
* made responsive using media queries
* also changed animation timing for different screen sizes
* note: behavior can be slightly un-aesthetic if the message is very long, but may not be a problem depending on usage?

### CODE REVIEW

File: react/frontpage/components/StreamMarquee.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <div className="marquee-container">
+        <div className="marquee">
+          <p>{this.props.message}</p>
+        </div>
+      </div>
+    );
+  }
+});
+
+module.exports = StreamMarquee;
```
Comment (binerys) [inline react/frontpage/components/StreamMarquee.jsx]: Add a new line here

File: react/frontpage/components/StreamMarquee.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    width: -moz-calc(100% - 160px - 100px);
+    width: -webkit-calc(100% - 160px - 100px);
+  }
+
+  .marquee p {
+    animation: marquee 5s linear infinite;
+    -moz-animation: marquee 5s linear infinite;
+    -webkit-animation: marquee 5s linear infinite;
+  }
+}
```
Comment (binerys) [inline react/frontpage/components/StreamMarquee.scss]: Add another new line here

Comment (jdliaw) [inline react/frontpage/components/StreamMarquee.scss]: Made all of the changes!

**Comment by binerys** [comment]
finally merging this in ✨

**Pull request merged**